src/scripts/OSFramework/OSUI/Pattern/Notification/Notification.ts

```diff
@@ -45,10 +47,8 @@ namespace OSFramework.OSUI.Patterns.Notification {
 		 * @memberof Notification
 		 */
 		private _autoCloseNotification(): void {
-			Helper.ApplySetTimeOut(() => {
-				if (this._isOpen) {
-					this.hide();
-				}
+			this._autoCloseTimeoutId = Helper.ApplySetTimeOut(() => {
+				this.hide();
```

Comment:
@JoaoFerreira-FrontEnd why have you removed the validation `if (this._isOpen)`?

Reply:
Forget it, I see you moved it to the `hide()` method

Reply:
It was already there.